inc/options/class-wpseo-option-wpseo.php

```diff
@@ -246,6 +247,7 @@ protected function validate_option( $dirty, $clean, $old ) {
 					break;
 				case 'previous_version':
 				case 'license_server_version':
+				case 'home_url':
```

Comment:
Should we use a more strict validation? This only checks if is set. In the main class there's a `validate_url()` method we already use, for example, for the social links.

Reply:
As this value is auto-generated, I'm not sure whether the extra validation is necessary.